auslib/web/public/client.py

```diff
@@ -89,7 +89,7 @@ def getQueryFromURL(url):
     query['osVersion'] = urllib.unquote(query['osVersion'])
     ua = request.headers.get('User-Agent')
     query['headerArchitecture'] = getHeaderArchitecture(query['buildTarget'], ua)
-    query['force'] = (int(query.get('force', 0)) == 1)
+    query['force'] = (query.get('force', 0) == "1")
```

Comment:
@hemant1612, glasserc is going to finish a new great feature for this flag in #382, "force" behavior will change a little and override this code region.